packages/core-data/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## v2.0.0 (Unreleased)
+
+- Breaking: `dispatch("core").receiveTerms` has been deprecated. Please use `dispatch("core").receiveEntityRecords` instead.
```

Comment:
It should be:

```md
## v2.0.0 (Unreleased)

### Breaking changes:

- `dispatch("core").receiveTerms` has been deprecated. Please use `dispatch("core").receiveEntityRecords` instead.
...
```
See https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs.

Reply:
I copied from another package :)

Reply:
Just saw that there's a bunch of package that don't follow the guidelines :) I'll update them as well.